### ISSUE TITLE
Use of @nuxtjs/prismic with nuxt-i18n

### ISSUE THREAD

**Issue by aceol** [nuxt-community/prismic-module#90]
In the newer version of [nuxt-18n](https://github.com/nuxt-community/nuxt-i18n), the url of the pages can be internationalized in the nuxt.config.js
```
  modules: [
    [
      'nuxt-i18n',
      {
        parsePages: false, // Required
        pages: {
          about: {
            'fr-fr': '/a-propos',
            'en-gb': '/about'
          }
        }
      }
    ],
```

If I miss `parsePage: false` it is yelling because it can't find preview.vue generated by @nuxtjs/prismic.

>  ERROR  ENOENT: no such file or directory, open '/Users/ace/Projects/PIX/pix-site/.nuxt/prismic/pages/preview.vue' 

But I need `parsePage: true`, because the pages name can also be specified in the page file

```
export default {
  name: 'About',
  nuxtI18n: {
    paths: {
          'fr-fr': '/a-propos',
          'en-gb': '/about'
    }
  },
```

Any advice on how to achieve that?

**Comment by lihbr** [comment]
Hey @aceol, sorry for the late answer, have you tried excluding `preview` from i18n: https://i18n.nuxtjs.org/routing.html#ignore-routes (last example), perhaps something like this could solve your issue, let me know!
```javascript
modules: [
  [
    'nuxt-i18n',
    {
      parsePages: false, // Required
      pages: {
        about: {
          'fr-fr': '/a-propos',
          'en-gb': '/about'
        }
      }
    }
  ],
  /* ... */
}
```

**Comment by angeloashmore** [comment]
@lihbr I just encountered this issue as well when using `@nuxtjs/i18n`.

Adding `pages.preview: false` fixes the issue, but only if `parsePages` is `false`.

```js
```

**Comment by lihbr** [comment]
Very nice catch! Didn't know about this option. Published in 1.4.2
